docs/src/_includes/layouts/base.html

```diff
@@ -13,11 +13,19 @@
         {% set page_title = site[hook].title %}
     {% endif %}
 
+    {% set rule_meta = rules_meta[title] %}
     {% set page_title = site.shared.title_format | replace("PAGE_TITLE", page_title) %}
-    {% set cover_image = ["https://", site.hostname, "/icon-512.png" ] | join %}
-    {% set cover_image_alt = site.shared.eslint_logo_alt %}
     {% set page_desc = site.shared.description %}
     {% set relative_page_url = page.url | url | prettyURL %}
+    {% set cover_image = [
+        "http://", "localhost:56606", "/og",
```

Comment:
I'm guessing that this shouldn't be `localhost`. Perhaps `site.hostname`?

Reply:
Sorry, I was using it in testing and forgot to change it back